JS-Assignment/Calculator/index.js

```diff
@@ -311,7 +301,6 @@ class Calculator {
 
 // class end ---------
 
-
 const input = document.getElementById('user-input')
 const output = document.getElementById('output');
```

Comment:
@bhavinbabariya Instead of declaring many different variables create one single variable combining all for html elements or direct assign the event listeners to the element.